charts/fediscoverer/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for fediscoverer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/mastodon/fediscoverer
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag:
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# App-specific configuration
+config:
+
+  # Fediverse secret key base.
+  # Can be provided as a secretKeyRef (recommended) or a string.
+  secretKeyBase:
+    secretKeyRef:
+      name:
+      key: secret-key-base
+
+  # PostgreSQL database connection information.
+  # Password can be provided as a secretKeyRef (recommended) or a string.
+  # Some values for the database are hard-coded into the app, and must be:
+  #   - Username: fediscoverer
+  #   - Databases (all owned by fediscoverer):
+  #     - fediscoverer_production
+  #     - fediscoverer_production_cache
+  #     - fediscoverer_production_queue
+  database:
+    host: localhost
+    port: 5432
+    password:
+      secretKeyRef:
+        name:
+        key: password
+
+serviceAccount:
```

Comment:
I do not think there is any use to create a service account for this app, it never talks to the Kubernetes API